tactical-microgrid-standard/controller/PowerDevicesRequestDataReaderListenerImpl.h

```diff
@@ -0,0 +1,19 @@
+#ifndef POWER_DEVICES_REQUEST_DATA_READER_LISTENER_IMPL_H
+#define POWER_DEVICES_REQUEST_DATA_READER_LISTENER_IMPL_H
+
+#include "CLIServer.h"
+#include "common/DataReaderListenerBase.h"
+
+class PowerDevicesRequestDataReaderListenerImpl : public DataReaderListenerBase {
+public:
+  explicit PowerDevicesRequestDataReaderListenerImpl(CLIServer& cli_server) : cli_server_(cli_server) {}
+
+  virtual ~PowerDevicesRequestDataReaderListenerImpl() {}
```

Comment:
```suggestion
  virtual ~PowerDevicesRequestDataReaderListenerImpl() = default;
```
Not sure it's any better though